docs/content/tools/prototyping.md

```diff
@@ -19,12 +19,14 @@ This method requires no dev environment set up and is useful for when you want t
   <head>
     <title></title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="stylesheet" href="https://unpkg.com/@primer/css@^16.0.0/dist/primer.css" />
+    <link rel="stylesheet" href="https://unpkg.com/@primer/css@^19.0.0/dist/primer.css" />
```

Comment:
Would it be possible to import the current version from `package.json` and interpolate it here?

Reply:
That would be great. Not sure if it's possible inside of these "code examples"? 🤔

Reply:
I've had a go and it seems you can if you use `<code>` instead of a md code snippet 🎉 

```jsx
<code class="language-html">
{`<!DOCTYPE html>
<html lang="en">
  <head>
    <title></title>
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <link rel="stylesheet" href="https://unpkg.com/@primer/css@^${packageJson.version}/dist/primer.css" />
  </head>
  <body></body>
</html>`}
</code>
```

Bit uglier but I think it's preferable to manually updating this page.

Reply:
Yep! This works:

![image](https://user-images.githubusercontent.com/378023/144522601-d64e5527-57c9-459b-93db-c28180ea10b1.png)

Awesome! 🙌  It did feel a bit like having to remember to update the year in the footer. 😄